src/booleans.js

```diff
@@ -1,61 +1,61 @@
 const negate = a => {
-  // your code here
+  return !a;
 };
 
 const both = (a, b) => {
-  // your code here
+  return (a && b);
 };
 
 const either = (a, b) => {
-  // your code here
+  return (a || b);
 };
 
 const none = (a, b) => {
-  // your code here
+  return (!a && !b);
 };
 
 const one = (a, b) => {
-  // your code here
+  return (a !== b);
 };
 
 const truthiness = a => {
-  // your code here
+  return !!a;
 };
 
 const isEqual = (a, b) => {
-  // your code here
+  return (a === b);
 };
 
 const isGreaterThan = (a, b) => {
-  // your code here
+  return (a > b);
 };
 
 const isLessThanOrEqualTo = (a, b) => {
-  // your code here
+  return (a <= b);
 };
 
 const isOdd = a => {
-  // your code here
+  return a % 2 !== 0;
 };
 
 const isEven = a => {
-  // your code here
+  return a % 2 === 0;
 };
 
 const isSquare = a => {
-  // your code here
+  return Number.isInteger(Math.sqrt(a));
 };
 
 const startsWith = (char, string) => {
-  // your code here
+  return string.charAt(0) === char;
```

Comment:
Have you noticed how strings behave in a similar way of arrays? They're like arrays of characters!